Scraper.py

```diff
@@ -160,10 +160,17 @@ def scrape_jobs(self):
                 "[position, company_name, company_url, date_ranges, job_location]);     }   } } return jobs; })();")
         except WebDriverException:
             jobs = []
-
+            
+        clean_jobs = []
+        for job in jobs: 
+            if job[2] != '':
+                clean_jobs.append(job)
+            elif job[2] == '': 
+                clean_jobs.append(['Fake', 'Fake', 'https://www.linkedin.com/company/linkedin/', '', ''])
```

Comment:
Why create a fake line? If you think job[2] == '' it's an edge case that should be excluded, than we could just exclude it.

```suggestion
            if job[2] != '':
                clean_jobs.append(job)
```